Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -6317,7 +6323,25 @@ static NSTextAlignment nsTextAlignmentFromTextAlignment(TextAlignment textAlignm
 
 #endif // ENABLE(IMAGE_ANALYSIS)
 
-bool WebViewImpl::imageAnalysisOverlayViewHasCursorAtPoint(NSPoint locationInView) const
+bool WebViewImpl::imageAnalysisOverlayViewShouldHandleMouseEvents(NSPoint locationInWindow) const
+{
+#if ENABLE(IMAGE_ANALYSIS_ENHANCEMENTS)
+    if (!m_imageAnalysisOverlayView)
+        return false;
+
+    auto interactionPoint = [m_view convertPoint:locationInWindow toView:m_imageAnalysisOverlayView.get()];
+
+    if (!CGRectContainsPoint(m_imageAnalysisInteractionBounds, interactionPoint))
+        return false;
+
+    return imageAnalysisOverlayViewHasInteractableItemAtPoint(interactionPoint) || [m_imageAnalysisOverlayView hasActiveTextSelection];
```

Comment:
Do you need an SPI declaration for `hasActiveTextSelection`?

Reply:
I didn’t think so since it’s API https://developer.apple.com/documentation/visionkit/imageanalysisoverlayview/4025534-hasactivetextselection , I’ll take a look and see what else could be wrong